plugins/cilium-cni/cni-install.sh

```diff
@@ -6,12 +6,22 @@ HOST_PREFIX=${HOST_PREFIX:-/host}
 CNI_CONF_NAME=${CNI_CONF_NAME:-10-cilium.conf}
 MTU=${MTU:-1450}
 
-CILIUM_CNI=${CILIUM_CNI:-${HOST_PREFIX}/opt/cni/bin/cilium-cni}
+CNI_DIR=${CNI_BINDIR:-${HOST_PREFIX}/opt/cni}
 CILIUM_CNI_CONF=${CILIUM_CNI_CONF:-${HOST_PREFIX}/etc/cni/net.d/${CNI_CONF_NAME}}
 
-# Install cilium-cni binary tohost
-echo "Installing $CILIUM_CNI ..."
-cp /opt/cni/bin/cilium-cni ${CILIUM_CNI}
+if [ ! -f ${CNI_DIR}/bin/loopback ]; then
+	echo "Installing loopback driver..."
+	mkdir tmp && cd tmp
+	wget https://storage.googleapis.com/kubernetes-release/network-plugins/cni-0799f5732f2a11b329d9e3d51b9c8f2e3759f2ff.tar.gz
+	mkdir -p ${CNI_DIR}/bin
+	sudo tar -xvf cni-0799f5732f2a11b329d9e3d51b9c8f2e3759f2ff.tar.gz
+	cp bin/loopback ${CNI_DIR}/bin/
```

Comment:
I thought about that but we have no clue what version is installed. It may be a custom version by the user. I found this to be the safest option.